release/models/rib/openconfig-rib-bgp-tables.yang

```diff
@@ -311,6 +321,639 @@ submodule openconfig-rib-bgp-tables {
     }
   }
 
+  grouping l2vpn-evpn-loc-rib-top {
+    description
+      "Top-level grouping for l2vpn evpn routing tables";
+
+    container loc-rib {
+      config false;
+      description
+        "Container for the l2vpn evpn BGP LOC-RIB data";
+
+      uses bgp-common-table-attrs-top;
+
+      container routes {
+        description
+          "Enclosing container for list of routes in the routing
+          table.";
+
+        list route-distinguisher {
+          description "List of route distinguishers";
+          key "route-distinguisher";
+
+          leaf route-distinguisher {
+            type leafref {
+              path "../state/route-distinguisher";
+            }
+            description
+              "Reference to the rd list key";
+          }
+
+          container state {
+            description "Top level containern for l2vpn evpn RDs";
+            leaf route-distinguisher {
+              type oc-ni-types:route-distinguisher;
```

Comment:
Should this leaf-ref the assigned RD of the EVPN?

Reply:
Discussed this with our team and this should not be a leaf-ref.  For all 3 network-instance cases/context (default NI, mac-vrf NI and ip-vrf NI) we need to support local and remote routes that will have different RDs.

Reply:
Clarify this in the description